packages/polaris-viz-core/src/constants.ts

```diff
@@ -338,12 +340,14 @@ export const LIGHT_THEME: Theme = {
     },
   },
   annotations: {
-    backgroundColor: variables.colorGray100,
-    textColor: variables.colorWhite,
-    titleColor: variables.colorWhite,
+    backgroundColor: 'rgb(227, 227, 227)',
```

Comment:
Do we want to stay consistent and add variables for these colors?

Reply:
I don't think so no. Once we move internally these are all going to be reworked anyways.